typescript/listeners/idle.py

```diff
@@ -162,15 +162,16 @@ def show_errors(self, diagno_event_body, syntactic):
                              sublime.DRAW_OUTLINED)
         else:
             settings = sublime.load_settings("TypeScript.sublime-settings")
+            error_color = settings.get("error_color") or "invalid.illegal"
+            error_icon = settings.get("error_icon") or ""
+            error_draw_style = \
+                sublime.DRAW_OUTLINED if settings.get("error_outlined") \
+                else sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE | sublime.DRAW_SQUIGGLY_UNDERLINE
             view.add_regions(region_key,
                              error_regions,
-                             settings.get("error_color", "invalid.illegal"),
-                             settings.get("error_icon", ""),
-                             sublime.DRAW_OUTLINED
-                             if settings.get("error_outlined") else
-                             sublime.DRAW_NO_FILL +
-                             sublime.DRAW_NO_OUTLINE +
```

Comment:
Who knows why this ever even worked.

Reply:
you've got me - this looks easier to reason about though 👍